rapthor/operations/image.py

```diff
@@ -175,6 +176,7 @@ def set_input_parameters(self):
                 nsubpipes = min(nsectors, nnodes)
                 nnodes_per_subpipeline = max(1, int(nnodes / nsubpipes) - 1)
                 self.input_parms.update({'mpi_nnodes': [nnodes_per_subpipeline] * nsectors})
+                self.input_parms.update({'mpi_ntasks_per_node': [self.parset['cluster_specific']['cpus_per_task']] * nsectors})
```

Comment:
I find the name `mpi_ntasks_per_node` a bit misleading. If I understand it correctly, then you create a list with `nsectors` integers that have the value `cpus_per_task`. I would assume then that the number of MPI tasks that can be run on a node is the quotient of `nproc` and `cpus_per_task`. Or am I wrong here?